docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - 9001:8080
 
   physical-address-annotator:
-    image: nlpsandbox/physical-address-annotator-example:0.3.1
+    image: nlpsandbox/physical-address-annotator-example:edge
```

Comment:
The `0.3.1` version tag of this annotator doesn't seem to implement the `tool` endpoint

Reply:
Strange. But yes, the endpoint `/tool` is in edge. I will release shortly a version 0.3.2 of the annotators shortly and then let you know when you can rely on the new versions.